src/contract_wrappers/token_registry_wrapper.ts

```diff
@@ -38,6 +47,44 @@ export class TokenRegistryWrapper extends ContractWrapper {
 
         const tokenRegistryContract = await this._getTokenRegistryContractAsync();
         const metadata = await tokenRegistryContract.getTokenMetaData.call(address);
+        const token = this._getTokenByMetadata(metadata);
+        return token;
+    }
+    public async getTokenAddressBySymbolIfExistsAsync(symbol: string): Promise<string|undefined> {
+        assert.isString('symbol', symbol);
+        const tokenRegistryContract = await this._getTokenRegistryContractAsync();
+        const addressIfExists = await tokenRegistryContract.getTokenAddressBySymbol.call(symbol);
+        if (addressIfExists === constants.NULL_ADDRESS) {
+            return undefined;
+        } else {
```

Comment:
Unnecessary if block here. Since the body of the `if` returns, we can simply return after the `if` block.